main/src/com/google/refine/expr/functions/FacetCount.java

```diff
@@ -74,14 +76,16 @@ public Object call(Properties bindings, Object[] args) {
 
                     column.setPrecompute(key, grouper);
                 } catch (ParsingException e) {
-                    return new EvalError("Error parsing facet expression " + facetExpression);
+                    // return new EvalError("Error parsing facet expression " + facetExpression);
+                    return new EvalError(EvalErrorMessage.fun_facet_count_error_parsing_facet(facetExpression));
                 }
             }
 
             return grouper.getChoiceValueCountMultiple(choiceValue);
         }
-        return new EvalError(ControlFunctionRegistry.getFunctionName(this) + 
-            " expects a choice value, an expression as a string, and a column name");
+        // return new EvalError(ControlFunctionRegistry.getFunctionName(this) +
+        //    " expects a choice value, an expression as a string, and a column name");
```

Comment:
Perhaps we can remove those comments now that the migration is made?

Reply:
Please excuse the delay in updating this. I hadnt't commited and synced the changes yet 🥲 I used [this ](https://github.com/elroykanye/mad-scripts/blob/master/openrefine_extraction_scripts/delete_comments/dc.py)to help out